include/mme-app/utils/mmeCauseTypes.h

```diff
@@ -26,7 +26,7 @@ typedef uint32_t MmeErrorCause;
 
 const MmeErrorCause noError_c = 0x00;
 const MmeErrorCause ueContextNotFound_c = 0x01;
-
+const MmeErrorCause secModeRespFailure_c = 0x02;
```

Comment:
This really needs cleanup...

Reply:
Can you clarify this comment? We can use generic error causes in mme-app and convert to protocol specific error causes as required before sending out the message. Does this approach need change?